src/OpenTelemetry/Metrics/Processors/PullMetricProcessor.cs

```diff
@@ -42,7 +42,7 @@ public void PullRequest()
             if (this.getMetrics != null)
             {
                 var metricsToExport = this.getMetrics(this.isDelta);
-                if (metricsToExport != null)
+                if (metricsToExport != null && metricsToExport.Metrics.Count > 0)
```

Comment:
there'd be more changes in the area, when we implement "clean-up" of aggregators if no new updates. (i think we currently send metricitem to exporters with zero as value)